### PR TITLE
Update Azure Container Apps Instruction

### PR DESCRIPTION
**Pull request by kimi-p**
- Single quote doesn't work since the string would contain `'` and breaks dd_site
- Traces are not sending to dd; only metrics and logs

### CODE REVIEW

File: content/en/serverless/azure_container_apps/_index.md [hunk 1]
```diff
@@ -200,13 +200,13 @@ az containerapp up \
...
   --target-port 80 \
-  --env-vars "DD_API_KEY=$DD_API_KEY" "DD_TRACE_ENABLED=true" "DD_SITE='datadoghq.com'" \
+  --env-vars "DD_API_KEY=$DD_API_KEY" "DD_TRACE_ENABLED=true" "DD_SITE=datadoghq.com" \
   --image YOUR_REGISTRY/YOUR_PROJECT
 ```
 
 ### 3. Results
 
-Once the deployment is completed, your metrics and traces are sent to Datadog. In Datadog, navigate to the [APM Trace Explorer][11] and search for your Container App by service name, or filter using the facet `origin:containerapp` to see your Azure Container App traces.
+Once the deployment is completed, your metrics and logs are sent to Datadog. In Datadog, navigate to the [APM Trace Explorer][11] and search for your Container App by service name, or filter using the facet `origin:containerapp` to see your Azure Container App traces.
```
Comment (IvanTopolcic) [inline content/en/serverless/azure_container_apps/_index.md:209]: Maybe we can add instructions to get tracing working, and/or link to the .NET tracing page?

Comment (kimi-p) [reply content/en/serverless/azure_container_apps/_index.md:209]: SG! I plan to add that after I can make tracing work just in case.

Comment (alexgallotta) [reply content/en/serverless/azure_container_apps/_index.md:209]: This change might not be accurate.
`logs` are not being exported with the extension but via
> To send logs without direct instrumentation, set up central [log collection through the Azure Integration](https://docs.datadoghq.com/integrations/azure/#log-collection), then use the [Diagnostic Settings on the Container App Environment](https://learn.microsoft.com/en-us/azure/container-apps/log-options#diagnostic-settings) to direct the logs to the configured logs pipeline.

Comment (IvanTopolcic) [reply content/en/serverless/azure_container_apps/_index.md]: Hmm not sure I follow @alexgallotta. Logs can be sent if the user enables `DD_LOGS_ENABLED=true`, in which case logs are sent directly via the `datadog-init` binary via stdout redirection. I think that's in the docs too?

Comment (IvanTopolcic) [reply content/en/serverless/azure_container_apps/_index.md]: Maybe we can add set `DD_LOGS_ENABLED` for logs (if you don't use the integration), and bring your own tracer (link to the .NET tracer page).

Comment (IvanTopolcic) [reply content/en/serverless/azure_container_apps/_index.md]: It also might make more sense to add instructions on getting the .NET tracer working in Container Apps/Cloud Run. I think that's definitely something we can improve on.

**Comment by apigirl** [comment]
Adding the WIP label. Please feel free to remove it when you're ready for docs team review!

File: content/en/serverless/azure_container_apps/_index.md (hunk 1 above)
Comment (kimi-p) [reply content/en/serverless/azure_container_apps/_index.md]: Danny already has another PR out for updating this doc. I'm closing this one in favor of his.

**Pull request closed without merging**